data_creation/incremental_data_creation/ads_data_generator.py

```
@@ -0,0 +1,300 @@
+    "flash_sale",
+]
+
+
+def generate_ads_data():
+    """Generate advertising data for all three platforms"""
+    generate_google_ads_data()
+    generate_facebook_ads_data()
+    generate_instagram_ads_data()
+
+
+def generate_google_ads_data():
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)